models/br_tse_eleicoes/schema.yml

```diff
@@ -1,583 +1,1219 @@
-        description: Entrega em conjunto
-  - name: br_tse_eleicoes__despesas_candidato
-    description: Microdados despesas candidato
-    columns:
-      - name: ano
-        description: Ano
-      - name: turno
-        description: Turno
-      - name: tipo_eleicao
-        description: Tipo da eleição
-      - name: sigla_uf
-        description: Sigla da unidade da federação
-      - name: id_municipio
-        description: ID Município - IBGE 7 Dígitos
-      - name: id_municipio_tse
-        description: ID Município - TSE
-      - name: numero_candidato
-        description: Número do candidato
       - name: cpf_candidato
         description: CPF do candidato
-      - name: sequencial_candidato
-        description: Sequencial do candidato
-      - name: id_candidato_bd
-        description: ID Candidato - Base dos Dados
-      - name: nome_candidato
-        description: Nome do candidato
+      - name: cpf_cnpj_fornecedor
+        description: CPF ou CNPJ do fornecedor
       - name: cpf_vice_suplente
         description: CPF do vice ou suplente
-      - name: numero_partido
-        description: Número do partido
-      - name: sigla_partido
-        description: Sigla do partido
-      - name: nome_partido
-        description: Nome do partido
-      - name: cargo
-        description: Cargo
-      - name: sequencial_despesa
-        description: Sequencial da despesa
       - name: data_despesa
         description: Data da despesa
-      - name: tipo_despesa
-        description: Tipo da despesa
-      - name: descricao_despesa
-        description: Descrição da despesa
-      - name: origem_despesa
-        description: Origem da despesa
-      - name: valor_despesa
-        description: Valor da despesa
-      - name: tipo_prestacao_contas
-        description: Tipo de prestação de contas
       - name: data_prestacao_contas
         description: Data de prestação de contas
-      - name: sequencial_prestador_contas
-        description: Sequencial do prestador de contas
-      - name: cnpj_prestador_contas
-        description: CNPJ do prestador de contas
-      - name: cnpj_candidato
-        description: CNPJ do candidato
-      - name: tipo_documento
-        description: Tipo do documento
-      - name: numero_documento
-        description: Número do documento
+      - name: descricao_cnae_2_fornecedor
+        description: Descrição da Classificação Nacional de Atividades Econômicas
+          2.0 do fornecedor
+      - name: descricao_despesa
+        description: Descrição da despesa
+      - name: esfera_partidaria_fornecedor
+        description: Esfera partidária do fornecedor
       - name: especie_recurso
         description: Espécie do recurso
       - name: fonte_recurso
         description: Fonte do recurso
-      - name: cpf_cnpj_fornecedor
-        description: CPF ou CNPJ do fornecedor
+      - name: id_candidato_bd
+        description: ID Candidato - Base dos Dados
+      - name: id_municipio
```

Comment:
teste id_municipio

Reply:
Excluí testes de `id_municipio` porque essa variável as vezes é nula. O TSE usa o `id_municipio_tse` para cidades fora do Brasil, que na hora de eu dar um merge com `id_municipio` elas ficam nulas.

Reply:
Ah, perfeito. 
Eu iria propor usar o macro de custom_relationships, mas se for muitas cidades, não vale a pena. 
Mas colocar isso na descrição é importante.

Reply:
Verdade vou anotar aqui. Tem uma série de pequenas melhorias nos metadados que já tenho listado para fazer.